sql.d/schema.sql

```diff
@@ -146,6 +146,7 @@ $$ LANGUAGE plpgsql;
 -- Main packages table
 --------------------------------------------------------------------------------
 CREATE TABLE packages (
+	id serial,
```

Comment:
Please use postgresql 10 new way of specifying identity columns, as here:

https://github.com/repology/repology/blob/694db62a67e517bef75e5ec5881973fbc4a394d0/sql.d/schema.sql#L428